openslides/mediafiles/static/js/mediafiles/image-plugin.js

```diff
@@ -0,0 +1,202 @@
+(function () {
+
+'use strict';
+
+angular.module('OpenSlidesApp.mediafiles.image-plugin', [])
```

Comment:
Please add dependencies. See other AngularJS modules of this app (it is not complete, I know).